### ISSUE TITLE
Theory - Neural Network

### ISSUE THREAD

**Issue by melanieselman** [Awesome-sw8-project/Report#56]
- [x] Neural Network Layer architecture (input, hidden, output layers, activation function)
- [x] Gradient Descent?
- [x] Backpropagation (loss function)
